### PR TITLE
Testing very basic fixtures

### PR DESCRIPTION
**Pull request by nerdstrike**

### CODE REVIEW

**Comment by nerdstrike** [comment]
@mksanger Review requested, I don't seem to have the privileges to assign you :/

**Comment by mksanger** [comment]
I don't have write access either, I would guess that you need to add the team to the repo somehow so that it gives everyone the same permissions as everywhere else

**Comment by nerdstrike** [comment]
> I don't have write access either, I would guess that you need to add the team to the repo somehow so that it gives everyone the same permissions as everywhere else

I think the permissions are fixed now. Let me know if you want to talk through what the repo (and the PR) are doing

**Review by mksanger — COMMENTED** [review]
We have been using pytest-it as a standard way to describe python tests, see npg_conda or partisan tests.

Other than that, the tests run, and I don't see any problems with the code.

File: setup.py [hunk 1]
```diff
@@ -22,9 +22,14 @@
...
         'pydantic',
+        'pysqlite',
         'psycopg2-binary',
         'sqlalchemy>=1.4.29',
         'ujson',
         'uvicorn'
+    ],
+    tests_require=[
+        'pytest',
+        'pytest-asyncio'
```
[the remark this answers is not in the document]
Comment (nerdstrike) [reply setup.py:33]: Iiiiinteresting

**Pull request merged**